test/Microsoft.ML.TestFrameworkCommon/Centrallizedlogger.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Data.SqlClient;
+
+namespace Microsoft.ML.TestFrameworkCommon
+{
+    public static class Centrallizedlogger
```

Comment:
❔ Is this used by local developer builds or only on CI?

Reply:
both

---
In reply to: [354509412](https://github.com/dotnet/machinelearning/pull/4510#discussion_r354509412) [](ancestors = 354509412)